dolphinscheduler-tools/src/main/java/org/apache/dolphinscheduler/tools/demo/DemoContants.java

```diff
@@ -0,0 +1,117 @@
+package org.apache.dolphinscheduler.tools.demo;
+
+public class DemoContants {
+    public static final String [] SHELL_taskDefinitionJson = {"[{\"code\":",
```

Comment:
Json string is hard to maintain, please replace it with `TaskDefinition` Object.